spyder_kernels/customize/spydercustomize.py

```diff
@@ -379,7 +323,7 @@ def get_current_file_name():
     try:
         return frontend_request().current_filename()
     except Exception:
-        _print("This command failed to be executed because an error occurred"
+        print("This command failed to be executed because an error occurred"
                " while trying to get the current file name from Spyder's"
                " editor. The error was:\n\n")
```

Comment:
```suggestion
              " while trying to get the current file name from Spyder's"
              " editor. The error was:\n\n")
```

To align these lines correctly.